content/en/blog/2023/jmx-metric-insight/index.md

```diff
@@ -0,0 +1,229 @@
+identification. JMX Metric Insight comes with a number of predefined
+configurations containing curated sets of JMX metrics for popular application
+servers or frameworks, such as:
+
+- [ActiveMQ](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/activemq.md)
+- [Hadoop](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/hadoop.md)
+- [Jetty](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/jetty.md)
+- [Kafka Broker](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/kafka-broker.md)
+- [Tomcat](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/tomcat.md)
+- [Wildfly](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/wildfly.md)
+
+You can also provide your own metric definitions, through one or more YAML
+files. The YAML file syntax documentation is available
+[here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics/javaagent#configuration-files).
+
+## Observe Kafka Broker metrics
+
+Let's observe the health of our Kafka Broker by exporting the predefined set of
+metrics using the JMX Metric Insight module and export it to Prometheus.
+
+Kafka can be installed on macOS using Homebrew with the following steps:
+
+```shell
+brew install kafka
+```
+
+To start Zookeeper:
+
+```shell
+zookeeper-server-start /usr/local/etc/kafka/zookeeper.properties
+```
+
+### Attach the OpenTelemetry java instrumentation agent
+
+Before starting the kafka broker, attach the OpenTelemetry java instrumentation
+agent to Kafka Broker by providing options in the KAFKA_OPTS environment
+variable. You can download the latest release of the agent from
+[here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases).
+
+```shell
+export KAFKA_OPTS="-Dapplication.name=my-kafka-app
+-Dotel.metrics.exporter=prometheus
+-Dotel.exporter.prometheus.port=9464
+-Dotel.service.name=my-kafka-broker
+-Dotel.jmx.target.system=kafka-broker
+-javaagent:/path/to/opentelemetry-javaagent.jar"
+```
+
+Now we can start the Kafka Broker:
+
+```shell
+kafka-server-start /usr/local/etc/kafka/server.properties
+```
+
+The Kafka broker should be up and running now. To test the installation, we can
+create a topic and use the Kafka console producer and console consumer. Create
+Kafka Topic:
+
+```shell
+kafka-topics --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 1 --topic my-test-topic
+```
+
+Start Kafka console producer to produce messages to the topic we created:
+
+```shell
+kafka-console-producer --broker-list localhost:9092 --topic test
+>First message
+>Second message
```

Comment:
If you're showing command output then use the `console` block language and include a command prompt, as shown below. Are the `>` actually part of the output?

```suggestion
```console
$ kafka-console-producer --broker-list localhost:9092 --topic test
>First message
>Second message
```

Reply:
Yes, the `>` are part of the Kafka ConsoleProducer output.